pavelib/utils/test/suites/nose_suite.py

```diff
@@ -190,7 +190,8 @@ def _default_test_id(self):
 
         if self.root == 'lms':
             default_test_id += " {system}/tests.py"
-            default_test_id += " openedx/core/djangolib"
+            default_test_id += " openedx/core/djangolib/*"
+            default_test_id += " openedx/features/*"
```

Comment:
@jzoldak See comments above.  Removing the `/*` after `openedx/features` resolves the problem, but is there a better way to tell nose that `README.rst` is not a test?